droid-src-sony-edo.spec

```diff
@@ -11,7 +11,7 @@
 %define ha_device_override sony-edo
 
 # repo service performed : %%include define-trees
-%define dhs_trees build development libcore platform_testing toolchain device libnativehelper tools art cts external vendor bionic dalvik frameworks sdk bootable hardware packages system developers kernel pdk test prebuilts/abi-dumps prebuilts/asuite prebuilts/build-tools prebuilts/bundletool prebuilts/checkcolor prebuilts/checkstyle prebuilts/clang-tools prebuilts/clang prebuilts/cmdline-tools prebuilts/devtools prebuilts/fuchsia_sdk prebuilts/gcc prebuilts/gdb prebuilts/go prebuilts/gradle-plugin prebuilts/jdk prebuilts/ktlint prebuilts/manifest-merger prebuilts/maven_repo prebuilts/misc prebuilts/ndk prebuilts/python prebuilts/r8 prebuilts/rust/linux-x86 prebuilts/sdk prebuilts/tools prebuilts/vndk
```

Comment:
If you are sure `prebuilts/{cmdline-tools,rust/linux-x86}` are both not needed to successfully build the image, then please remove them from https://github.com/mer-hybris/droid-src-sony/blob/e88e4975694119c0cdf21605ecf3c4385de453e5/source.paths#L33 and add to this PR.
Originally they were added to for Android 10: https://github.com/mer-hybris/droid-src-sony/commit/e88e4975694119c0cdf21605ecf3c4385de453e5#diff-e3b60c47a587e4da528a6eb288dc43d42cdc1367639c82c0d7b8e1ffc8cc02d9

Reply:
> > Originally they were added to for Android 11: [e88e497#diff-e3b60c47a587e4da528a6eb288dc43d42cdc1367639c82c0d7b8e1ffc8cc02d9](https://github.com/mer-hybris/droid-src-sony/commit/e88e4975694119c0cdf21605ecf3c4385de453e5#diff-e3b60c47a587e4da528a6eb288dc43d42cdc1367639c82c0d7b8e1ffc8cc02d9)

> I build hal multiple times with these changes so I assume this is enough, will do so.

`source.paths` and .spec files packs source RPMs to be built on OBS. After testing locally, it doesn't build without the two directories:
```
$ mv prebuilts/{cmdline-tools,rust/linux-x86} .. -v
$ make -j8 systemimage vendorimage
...
FAILED: out/soong/build.ninja
out/soong/.bootstrap/bin/soong_build -t -l out/.module_paths/Android.bp.list -b out/soong -n out -d out/soong/build.ninja.d -globFile out/soong/.bootstrap/bui
ld-globs.ninja -o out/soong/build.ninja Android.bp
error: external/rust/crates/unicode-xid/Android.bp:15:1: "unicode-xid_device_tests_unicode_xid" depends on undefined module "libstd.static"
error: external/rust/crates/unicode-xid/Android.bp:15:1: "unicode-xid_device_tests_unicode_xid" depends on undefined module "libtest.static"
error: external/rust/crates/unicode-xid/Android.bp:15:1: "unicode-xid_device_tests_unicode_xid" depends on undefined module "libterm.static"
error: external/rust/crates/unicode-xid/Android.bp:15:1: "unicode-xid_device_tests_unicode_xid" depends on undefined module "libstd"
error: external/rust/crates/unicode-xid/Android.bp:15:1: "unicode-xid_device_tests_unicode_xid" depends on undefined module "libtest"
22:51:49 soong bootstrap failed with: exit status 1

#### failed to build some targets (43 seconds) ####
```

After replacing rust compiler back, the build fails further, due to then `cmdline-tools` missing.

Reply:
Sorry I ran the script in the wrong tree, I assumed the change was intentional but didn't pay attention.

Should be fixed now.